data/fda_adverse_reactions/transform.py

```diff
@@ -0,0 +1,126 @@
+import concurrent.futures
+import os
+from xml.etree import ElementTree
+
+import pandas as pd
+import requests
+import yaml
+
+DATASET_URL = "ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/23.02/output/etl/json/fda/significantAdverseDrugReactions"  # noqa
+DOWNLOAD_FOLDER = "./fda/significantAdverseDrugReactions"
+EBI_URL = "https://www.ebi.ac.uk/chembl/api/data/molecule/{}"
+
+META_TEMPLATE = {
+    "name": "fda_adverse_reactions",  # unique identifier, we will also use this for directory names
+    "description": "A dataset of adverse reaction statistics for drugs and reaction events.",
+    "targets": [
+        {
+            "id": "count",  # name of the column in a tabular dataset
+            "description": "A count of how many reaction events occurred for this chembl id.",
+            "units": None,  # units of the values in this column (leave empty if unitless)
+            "type": "ordinal",  # can be "categorical", "ordinal", "continuous", "string"
+            "names": [  # names for the property (to sample from for building the prompts)
+                {"noun": "adverse reaction frequency"},
```

Comment:
If the units are none then it would be better to use the word "count" instead of "frequency" (but maybe I'm biased by me electronics background where frequency is always 1/count).